modules/gdscript/gdscript_editor.cpp

```diff
@@ -2953,7 +2953,7 @@ Error GDScriptLanguage::complete_code(const String &p_code, const String &p_path
 
 #else
 
-Error GDScriptLanguage::complete_code(const String &p_code, const String &p_path, Object *p_owner, List<String> *r_options, bool &r_forced, String &r_call_hint) {
+Error GDScriptLanguage::complete_code(const String &p_code, const String &p_path, Object *p_owner, List<ScriptCodeCompletionOption> *r_options, bool &r_forced, String &r_call_hint) {
```

Comment:
This and the above change seem to be fixups of the first commit, it would be best if they could be amended directly in the first commit, keeping the second solely about icons. Especially since I think the second commit will need more work and discussion given how it impacts TextEdit (might be worth splitting the two in different PRs, so that we can merge the better code completion first, and review the icons feature in a second step).

Reply:
> might be worth splitting the two in different PRs, so that we can merge the better code completion first, and review the icons feature in a second step
That sounds good.
But I'd like to know how can I make the second PR before the first one is merge ?

Reply:
You can make the second PR including both commits (the first one which will also be in its own PR, and the second one). Once the first PR is merged, you can then rebase on master to remove the first commit from that second PR.